packages/cactus-plugin-cc-tx-visualization/src/test/csv/dummy-use-case-6-events.csv

```diff
@@ -0,0 +1,7 @@
+caseID;receiptID;timestamp;blockchainID;invocationType;methodName;parameters;identity;cost;carbonFootprint;latency;revenue
+1;txID1;2024-04-15T18:11:42.596Z;FABRIC_2;FabricContractInvocationType.SEND;InitializeAsset;"[""asset1"",""100""]";A;0;0.00018;39;0
+1;txID2;2024-04-15T18:11:42.598Z;FABRIC_2;FabricContractInvocationType.SEND;LockAsset;"[""asset1""]";A;0;0.00018;39;0
+1;txID3;2024-04-15T18:11:42.599Z;FABRIC_2;FabricContractInvocationType.SEND;CreateAsset;"[""1"",""100""]";A;0;0.00018;60;0
+1;txID4;2024-04-15T18:11:42.600Z;FABRIC_2;FabricContractInvocationType.SEND;TransferAsset;"[""asset1"",""targetAccount""]";A;0;0.00018;61;0
+1;txID5;2024-04-15T18:11:42.601Z;FABRIC_2;FabricContractInvocationType.SEND;TransferAsset;"[""asset1"",""targetAccount2""]";A;0;0.00018;61;0
```

Comment:
have these tests all start with `example-dummy`. don't forget to change the file name on the csv creation logic in the tests